src/components/featuredContent/featuredContent.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  FeaturedContent,
+  Button,
+  Heading,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import featuredContentData from "@/data/featuredContentData";
+
+const FeaturedContentComponent = () => {
+  var randomNumber = Math.random() < 0.5 ? 0 : 1;
```

Comment:
It'd be good to have a few simple tests for this component but this random function will make it tricky to test. You can add a `{ testRandomNumber }` prop (for example), and use that in your test as `render(<FeaturedContent testRandomNumber={0} />)`.

And here you can check if `testRandomNumber` was passed and if not then run the code you have to generate the random number. It's a somewhat typical pattern to pass test-only values or props down to functions or components when you need it to be a certain value only for testing purposes.